### ISSUE TITLE
RBF Transactions

### ISSUE THREAD

**Issue by 555Saitama** [mycitadel/mycitadel-desktop#105]
Hey guys,

Is it possible to add a RBF option when creating transactions ? 
For a multi sig wallet in particular, there can be some delay between the preparation of the transaction and its actual broadcast.
Thus, the fees opted for at the beginning may no longer be pertinent.

Thanks

**Comment by dr-orlovsky** [comment]
Yep, this was on the roadmap, and I will prioritize that. We also will do "cancel transactions" (RBF paying to own change address).
